inst/htmljs/animint.js

```diff
@@ -1080,7 +1080,19 @@ var animint = function (to_select, json_file) {
       }
       return size;
     };
-
+    
+    // stroke_width for geom_point
+    var stroke_width = 0.5;  // by default ggplot2 has 0.5
```

Comment:
the previous default in animints seems to be stroke_width=1 so can we please keep that?
![screenshot-stroke-width](https://cloud.githubusercontent.com/assets/932850/16732809/e42f5a20-474c-11e6-851b-747f5b9b01ac.png)